dev/releases/utils.py

```diff
@@ -15,42 +15,54 @@
 import subprocess
 import sys
 
+
 # print notices in green
 def notice(msg):
     print("\033[32m" + msg + "\033[0m")
 
+
 # print warnings in yellow
 def warning(msg):
     print("\033[33m" + msg + "\033[0m")
 
+
 # print error in red and exit
 def error(msg):
     print("\033[31m" + msg + "\033[0m")
     sys.exit(1)
 
+
 def verify_command_available(cmd):
-    if shutil.which(cmd) == None:
+    if shutil.which(cmd) is None:
         error(f"the '{cmd}' command was not found, please install it")
     # TODO: do the analog of this in ReleaseTools bash script:
     # command -v curl >/dev/null 2>&1 ||
     #     error "the 'curl' command was not found, please install it"
 
+
 def verify_git_repo():
-    res = subprocess.run(["git", "--git-dir=.git", "rev-parse"], stderr = subprocess.DEVNULL)
+    res = subprocess.run(
+        ["git", "--git-dir=.git", "rev-parse"], stderr=subprocess.DEVNULL, check=False
+    )
     if res.returncode != 0:
         error("current directory is not a git root directory")
 
+
 # check for uncommitted changes
 def is_git_clean():
-    res = subprocess.run(["git", "update-index", "--refresh"])
+    res = subprocess.run(["git", "update-index", "--refresh"], check=False)
     if res.returncode == 0:
-        res = subprocess.run(["git", "diff-index", "--quiet", "HEAD", "--"])
+        res = subprocess.run(
+            ["git", "diff-index", "--quiet", "HEAD", "--"], check=False
+        )
```

Comment:
What are you using to format the code? I just tried `black` and it doesn't touch `is_git_clean` for me

Reply:
I used black, sorry, I didn't really mean to format everything again. Note that here I also added `check=False`, which might make the difference.